File: ProgettoMobidev/app/src/main/java/com/capraraedefrancescosoft/progettomobidev/models/FacebookFriend.java
package com.capraraedefrancescosoft.progettomobidev.models;

/**
 * Created by Gianpaolo Caprara on 9/13/2016.
 */
public class FacebookFriend {
    private String name;
    private String id;
    private boolean checked;

    public FacebookFriend(String name, String id){
        this.id = id;
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public boolean isChecked() {
        return checked;
    }

    public void setChecked(boolean checked) {
        this.checked = checked;
    }
}

File: ProgettoMobidev_iOS/progettomobidev/FacebookFriend.m
//
//  FacebookFriend.m
//  Travel Notes
//
//  Created by gianpaolo on 06/10/16.
//  Copyright © 2016 capraraedefrancescosoft. All rights reserved.
//

#import "FacebookFriend.h"

@implementation FacebookFriend

// getter e setter
@synthesize name,idFriend,checked;

// implementazione dei costruttori
- (FacebookFriend*) initWithName: (NSString*) newName andIdFriend: (NSString*) newIdFriend {
    self = [super init];
    if (self){
        name = newName;
        idFriend = newIdFriend;
    }
    return self;
}

@end